### ISSUE TITLE
Unable to run url_decode

### ISSUE THREAD

**Issue by jakemcgraw** [colinta/SublimeStringEncode#21]
System

- Sublime Text 3
- OS X 10.10
- Installed through PackageControl on 05/19/2015

Repro

1. Select URL encoded text
2. Select Edit > StringEncode > URL Decode
3. Receive following output in ST console:

```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 556, in run_
    return self.run(edit)
  File "string_encode in /Users/jake/Library/Application Support/Sublime Text 3/Installed Packages/StringEncode.sublime-package", line 33, in run
  File "string_encode in /Users/jake/Library/Application Support/Sublime Text 3/Installed Packages/StringEncode.sublime-package", line 165, in encode
  File "./urllib/parse.py", line 615, in unquote_plus
TypeError: expected bytes, bytearray or buffer compatible object
```

**Comment by colinta** [comment]
You're right!  This regression is from ST2 support I think.  I found a quick fix.  Thanks for reporting!

**Comment by colinta** [comment]
Fixed with 13265e2, available in 2.0.12
